test/Microsoft.ML.Benchmarks/Helpers/BenchmarkHelpers.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace Microsoft.ML.Benchmarks
+{
+    internal class BenchmarkHelpers
```

Comment:
this name is too general. What this class does? can you descibe it using simple words? these words should be the name of the class